internal/core/services/room_manager/room_manager.go

```diff
@@ -82,6 +85,18 @@ func (m *RoomManager) CreateRoom(ctx context.Context, scheduler entities.Schedul
 		return nil, nil, err
 	}
 
+	// TODO: let each scheduler parametrize its timeout and use this config as fallback if the scheduler timeout value
```

Comment:
@gabrielcorado @luizmiranda7 What do you guys think about this idea? the initialization timeout value of the scheduler's containers will always be the same? would it make sense to be able to define this value for each individual scheduler?

Reply:
By now I don't see the need of having it for each scheduler, we can watch it in production later on. Basically, we can have a high configured timeout to hold every room addition.